doc/tutorial/profiling.txt

```diff
@@ -33,11 +33,12 @@ functions using either of the following two options:
     - You can also combine the profile of many functions: 
     
       .. testcode::
+          :hide:
 
           profile = theano.compile.ProfileStats()
-          f = theano.function(..., profile=profile)
-          g = theano.function(..., profile=profile)
-          ...
+          f = theano.function(..., profile=profile)  # doctest: +SKIP
+          g = theano.function(..., profile=profile)  # doctest: +SKIP
+          ...  # doctest: +SKIP
```

Comment:
Adding a blank line still doesn't solve the problem. What we need here is to have a testcode block that doesn't actually get executed. I also tried adding "# doctest: +skip" at the end of each line, but it didn't work too.